pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
```

Comment:
Same here and for `turbo.json`, unless we plan to use turborepo for the CLI in some capacity